<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1318</width>
    <height>1012</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QWidget" name="widget" native="true">
      <property name="minimumSize">
       <size>
        <width>500</width>
        <height>350</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>500</width>
        <height>350</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QWidget" name="widget_2" native="true">
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="3">
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>42</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="2">
        <widget class="QPushButton" name="select_img_btn">
         <property name="text">
          <string>选择</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0" colspan="2">
        <widget class="ShotPolygonLabel" name="img_lab_2">
         <property name="minimumSize">
          <size>
           <width>480</width>
           <height>320</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>480</width>
           <height>320</height>
          </size>
         </property>
         <property name="text">
          <string>rolygon</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="img_path"/>
       </item>
       <item row="0" column="7">
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="3" column="0" colspan="2">
        <widget class="QLabel" name="label_3">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>30</height>
          </size>
         </property>
         <property name="text">
          <string>位置信息：</string>
         </property>
        </widget>
       </item>
       <item row="0" column="4">
        <widget class="QPushButton" name="rect_btn">
         <property name="text">
          <string>画矩形</string>
         </property>
        </widget>
       </item>
       <item row="1" column="6">
        <widget class="ShotMergeLabel" name="img_merge_lab">
         <property name="minimumSize">
          <size>
           <width>480</width>
           <height>320</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>480</width>
           <height>320</height>
          </size>
         </property>
         <property name="text">
          <string>merge</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0" colspan="2">
        <widget class="ShotLabel" name="img_lab">
         <property name="minimumSize">
          <size>
           <width>480</width>
           <height>320</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>320</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>rect</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>图片</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0" colspan="3">
        <widget class="QTextEdit" name="pos_text">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>100</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="0" column="5">
        <widget class="QPushButton" name="polygon_btn">
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>画多边形</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1318</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ShotLabel</class>
   <extends>QLabel</extends>
   <header>LhtGui/DrawLine/shot_label.h</header>
  </customwidget>
  <customwidget>
   <class>ShotPolygonLabel</class>
   <extends>QLabel</extends>
   <header>LhtGui/DrawLine/shot_polygon_label.h</header>
  </customwidget>
  <customwidget>
   <class>ShotMergeLabel</class>
   <extends>QLabel</extends>
   <header>LhtGui/DrawLine/shot_merge_label.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
